### ISSUE TITLE
Findability of emojis

### ISSUE THREAD

**Issue by muffin-p** [Merrit/feeling_finder#106]
I am not sure how the search works, so I apologize if this isn't an issue with the app itself. It would be really nice :
1) If the application had additional tags for emojis Or
2) The application allowed the users to import their own custom tags for emojis.

Ex: LOL gives me a lollipop in the search results when more than likely I am just looking for laughing emoji(s), so having the laughing emojis tagged with LOL would be a really nice feature.

**Comment by Merrit** [comment]
This sort of association would have to be added manually, as the emoji dataset doesn't have any such tags or groupings.

Using the `lol` example, I could add an association that if the search term is `lol` then these emojis should be added to the results: 😆 😂 🤣

I can see the utility; I would use that one myself. It would be too much to add a whole bunch of such manual associations though. Aside from the `lol` one, was there something specific you were hoping for?

**Comment by muffin-p** [comment]
Yes, ability to manually associate keywords to emojis via configuration files would be awesome too, and then I can import them on my android app and also Linux app. 

There are instances where search doesn't work well. Just a few of them at the top of my head:
1. I can never find the Vulcan salute 🖖🏽. (In fact I don't even know the keyword for it. A reverse lookup dictionary of emoji to keyword would be quite beneficial to just train end users.)
2. Country names, ex: UAE should give me 🇦🇪, but I must search for United...
3. Namaste (Indian word for salutation) should give me 🙏🏽

**Comment by Merrit** [comment]
On desktop, if you hover over an emoji it shows its descriptive name. For example:

<img src="https://github.com/user-attachments/assets/fd400ed7-d730-465e-9cdf-079f7d53b42c" width="80%">

With this I know I can search for something like `raised middle` to get this result. That said, I can definitely see how it would be easier to remember `vulcan` if you had made that association. Also, viewing the text doesn't seem to be something I added to the Android build!

I have a couple of ideas for things that might work, though I am really busy at present. I will mull them over, and see what we can come up with. :)

**Comment by muffin-p** [comment]
No problem at all! Your time is appreciated. Also just realized you are in Canada too - Greetings from Nova Scotia :-).
